zbench.zig

```diff
@@ -317,7 +327,7 @@ pub fn run(comptime func: BenchFunc, bench: *Benchmark, benchResult: *BenchmarkR
     });
 
     bench.setTotalOperations(bench.N);
-    bench.report();
 
+    try bench.report();
```

Comment:
I don't think we need to call `Benchmark.report` anymore now that `Benchmark.prettyPrint` displays the number of runs performed in the pretty-table

Reply:
Right, good point. Should we rename `prettyPrint` --> `report` then ?

Reply:
implemented via https://github.com/hendriknielaender/zBench/pull/34/commits/4a8462b51f8973c80cc4142c81ffe5c066cdafb7